### DISCUSSION TITLE
The USWDS Should Apply as a Digital Public Good

### DISCUSSION THREAD

**Discussion by mgifford** [uswds/uswds#6960] [Ideas]
There is a growing movement to encourage government open source projects to move beyond simply being open source, but specifically aligning to the public good.

https://en.wikipedia.org/wiki/Digital_public_goods

To be a registered Digital Public Good (DPG) there is an application process:

https://www.digitalpublicgoods.net/registry

This registry is supported by USAID. 

It would be great to have the USWDS recognized.  Certainly, this is a design system used outside the USA Federal Government.

**Comment by amycole501** [comment]
Conducting research into the pros and cons as well as starting the application process to determine if we are able to become part of the DPG registry.

**Comment by amycole501** [comment]
Review is finished. @annepetersen has quite a bit of experience with this. For now we are moving this to the backlog as we gather more information.

**Comment by annepetersen** [comment]
Hey @mgifford: this isn't something the day-to-day team can take on right now, but could potentially delegate to our open source community, if you'd like to start the process

**Comment by ryparker** [comment]
I've moved this to Ideas so the community can continue the proposal. As the team explained in August, it can't take on the application right now but welcomes a community-led effort. The research and offer to help are still useful; registration hasn't been completed, and the idea hasn't been rejected.

If you'd like to help, please share which parts you can take on, the application requirements you've checked, and any decisions you need from maintainers. Once there's an owner and an agreed next step, we can open an implementation task. The original proposal and comments are all preserved here.
